packages/twenty-server/src/modules/calendar/standard-objects/calendar-channel.workspace-entity.ts

```diff
@@ -39,6 +39,13 @@ export enum CalendarChannelSyncStage {
   FAILED = 'FAILED',
 }
 
+export enum CalendarChannelContactAutoCreation {
```

Comment:
I feel we are the naming could be improved
CalendarChannelContactAutoCreationPolicy or CalendarChannelContactAutoCreationMode

Reply:
I agree